tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1532,4 +1532,84 @@ public function theResponseShouldContainTheFollowingDriveInformation(TableNode $
 			throw new Error('Response is not an array or the array does not consist key "drive"');
 		}
 	}
+
+	/**
+	 * @When user :user gets details of a group :groupName using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $groupName
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfAGroupUsingTheGraphApi(string $user, string $groupName): void {
+		$groupId = $this->getGroupIdByName($groupName);
+		$credentials = $this->getAdminOrUserCredentials($user);
+
+		$this->featureContext->setResponse(
+			GraphHelper::getSingleGroup(
+				$this->featureContext->getBaseUrl(),
+				$groupId,
+				$this->featureContext->getStepLineRef(),
+				$credentials["username"],
+				$credentials["password"]
+			)
+		);
+	}
+
+	/**
+	 * Get Group Id from response
+	 *
+	 * @param string $groupName - name of the group
+	 * @param array|null $allGroups
+	 *
+	 * @return string - id of the group
+	 * @throws Exception | GuzzleException
+	 */
+	public function getGroupIdByName(string $groupName, ?array $allGroups = null): string {
```

Comment:
I think this method is implemented.
upss my fail. it is only in the master https://github.com/owncloud/ocis/pull/5575/files

Reply:
I could not move my PR to stable branch because there is no code "filtering feature"

Reply:
It could be great if we can port that code changes to the stable as well.

Reply:
Thanks :heart:

Reply:
I want to get methods `getGroupIdByName ` and `theJsonRespondedShouldContainAGroupWithTheseKeyAndValuePairs` from master. But the further I go, the more dependencies I have

Reply:
I didn't see `theJsonRespondedShouldContainAGroupWithTheseKeyAndValuePairs` in the master. where was it added before?

Reply:
done